NetKAN/KSP-Recall.netkan

```diff
@@ -4,7 +4,7 @@
     "$vref"            : "#/ckan/ksp-avc",
     "identifier"       : "KSP-Recall",
     "name"             : "KSP Recall",
-    "abstract"         : "KSP Recall - a bag of tricks and stunts to work around KSP bugs while editing Resources on Editor, and more to come...",
+    "abstract"         : "KSP Recall - a bag of tricks and stunts to work around KSP bugs. Currently soves the Resource resetting on KSP 1.9.x, the heading drift on grounded crafts on KSP >= 1.8, and more to come...",
```

Comment:
```suggestion
    "abstract"         : "KSP Recall - Solves the resource resetting on KSP 1.9.x, the heading drift on grounded crafts on KSP >= 1.8, and more to come...",
```

"Solves" is misspelled, and we might as well let the reader get to the helpful text as quickly as possible.

Reply:
Accepted. Thanks for the tip.